cmake/modules/FindZephyr-sdk.cmake

```diff
@@ -60,7 +60,44 @@ if(("zephyr" STREQUAL ${ZEPHYR_TOOLCHAIN_VARIANT}) OR
       set(ZEPHYR_TOOLCHAIN_VARIANT ${ZEPHYR_CURRENT_TOOLCHAIN_VARIANT})
     endif()
   else()
-    find_package(Zephyr-sdk ${Zephyr-sdk_FIND_VERSION} REQUIRED QUIET CONFIG PATHS
+    # Search for Zephyr SDK version 0.0.0 which does not exist, this is needed to
+    # return a list of compatible versions and find the best suited version that
+    # is available.
+    find_package(Zephyr-sdk 0.0.0 EXACT QUIET CONFIG PATHS
+                 /usr
+                 /usr/local
+                 /opt
+                 $ENV{HOME}
+                 $ENV{HOME}/.local
+                 $ENV{HOME}/.local/opt
+                 $ENV{HOME}/bin)
+
+    set(PACKAGE_FIND_VERSION ${Zephyr-sdk_FIND_VERSION})
```

Comment:
This is something that `find_package()` itself takes care of, I really don't think we should start re-implementing a scaled down version of how `find_package()` works in this case.

Reply:
Fixed